appcues/src/main/java/com/appcues/data/model/ExperienceStepFormState.kt

```diff
@@ -18,6 +18,9 @@ internal class ExperienceStepFormState {
     val formItems: Collection<ExperienceStepFormItemState>
         get() = _formItems.values.sortedBy { it.index }
 
+    val lastTextInputItem: ExperienceStepFormItemState?
+        get() = _formItems.values.lastOrNull { it.isTextFocusable }
```

Comment:
you might want to use `formItems` (the index sorted list) rather than `_formItems.values` (potentially not sorted)? to be safe